apps/web/src/components/layout/components/SideNav.tsx

```diff
@@ -62,8 +63,8 @@ export function SideNav({}: Props) {
   const dark = colorScheme === 'dark';
   const { addItem, removeItems } = useSpotlightContext();
   const { classes } = usePopoverStyles();
-  const isMultiTenancyEnabled = useIsMultiTenancyEnabled();
-  const isTranslationManagerEnabled = useIsTranslationManagerEnabled();
+  const isMultiTenancyEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_MULTI_TENANCY_ENABLED);
+  const isTranslationManagerEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_TRANSLATION_MANAGER_ENABLED);
```

Comment:
Using the refactored `useFeatureFlag` in UI, only key provision is necessary. No new functions needed.